CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Upgrade Cilium to [v1.16.0](https://github.com/cilium/cilium/releases/tag/v1.16.0).
+- Disable digest in all images.
+- Remove metrics-port patch.
```

Comment:
```suggestion
```

I'd say it isn't relevant for the customers. I'm also not sure about the digest. They are not impacted in any way I don't think.